src/components/common-components/LargeTextBlock/LargeTextBlock.js

```diff
@@ -0,0 +1,50 @@
+import React, {useState} from 'react';
+import './LargeTextBlock.css';
+import Fade from 'react-reveal/Fade';
+
+/**
+ * @returns {React.Component}
```

Comment:
nit JSDoc should have more detail about the expected inputs and outputs. Because this project is not using typescript, these are even more important to document the inputs